From: Debian Multimedia Maintainers
 <pkg-multimedia-maintainers@lists.alioth.debian.org>
Date: Thu, 16 Mar 2017 20:48:20 +0100
Subject: _fix_sh772x_build

Fix FTBFS on Renesas SH (sh4) because libjpeg doesn't link against sh772x driver.
See http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=536314
---
 gfxdrivers/sh772x/Makefile.am | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/gfxdrivers/sh772x/Makefile.am b/gfxdrivers/sh772x/Makefile.am
index f1bc645..ac46695 100644
--- a/gfxdrivers/sh772x/Makefile.am
+++ b/gfxdrivers/sh772x/Makefile.am
@@ -22,6 +22,8 @@ INCLUDES = \
 	-I$(top_srcdir)/systems		\
 	-I$(srcdir)/kernel-module
 
+libsh7722_jpeg_la_LIBADD = \
+        $(LIBJPEG)
 
 sh7722_LTLIBRARIES = libdirectfb_sh7722.la
 
